modules/compiler/utils/source/work_item_loops_pass.cpp

```diff
@@ -1714,7 +1714,9 @@ Function *compiler::utils::WorkItemLoopsPass::makeWrapperFunction(
 
   // Remap any constant expression which take a reference to the old function
   // FIXME: What about the main function?
-  for (auto *user : refF.users()) {
+  for (auto it = refF.user_begin(); it != refF.user_end();) {
```

Comment:
I've never used it myself but I wonder if this is what `llvm::make_early_inc_range` is for?